src/FirstSteps/RPG/Adventures/Hell.cs

```diff
@@ -0,0 +1,68 @@
+﻿using FirstSteps.RPG.Heroes;
+using FirstSteps.RPG.Tools;
+using Spectre.Console;
+using System;
+
+namespace FirstSteps.RPG.Adventures
+{
+    public class Hell
+    {
+        public void FightWithboss(Hero hero)
+        {
+            Boss boss = new Boss("Diablo")
+            {
+                Health = 55,
+                Damage = 6,
+                Strength = 4,
+            };
+            Display.DefaultText($"You entered to the Hell!");
+
+            while (boss.Health >= 0)
+            {
+                string userInput = AnsiConsole.Prompt(
+                    new SelectionPrompt<string>()
+                      .Title($" \nChose what u want to do! {Emoji.Known.CrossedSwords}")
+                      .AddChoices("Boss stats", "Hit the boss!", "Run"));
+
+                switch (userInput)
+                {
+                    case "Hit the boss!":
+                        var number = new Random().Next(1, 5);
+                        Console.WriteLine($"You rolled {number}");
+                        if (number > 2)
+                        {
+                            boss.Health -= hero.DealDamage();
```

Comment:
You always do the `DealDamage` calculation twice. What if it was an expensive operation took 5s?

Reply:
Also, it would be better to add `TakeDamage` to the boss.